modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedAndesContextStoreImpl.java

```diff
@@ -195,6 +196,14 @@ private void createSchema() {
      * {@inheritDoc}
      */
     @Override
+    public Map<String, String> getAllDurableSubscriptionsByID() throws AndesException {
+        throw new NotImplementedException();
```

Comment:
Why are we using sun packages? This is an internal proprietary API. Better to use java.lang.UnsupportedOperationException